Sources/SIL/SILParser.swift

```diff
@@ -199,6 +218,7 @@ class SILParser: Parser {
         case "store":
             let value = try parseValue()
             try take("to")
+            let _ = skip("[init]") // Used in ownership SSA
             let _ = skip("[trivial]") // Used in ownership SSA
```

Comment:
Likewise, let's add these flags to fields of `store`.

Reply:
Hmm I didn't want to pollute the instruction with too many fields, and I don't know whether they're exclusive or not, so I'm not sure whether we can make that an enum too. This stuff is entirely undocumented and so I ignored it. I can add it though if you really want it.